webapp/src/ts/services/search.service.ts

```diff
@@ -122,11 +128,23 @@ export class SearchService {
       });
   }
 
-  search(type, filters: Filter, options:any = {}, extensions:any = {}, docIds: any[] | undefined = undefined) {
-    return this.ngZone.runOutsideAngular(() => this._search(type, filters, options, extensions, docIds));
+  search(type, filters: Filter, options:SearchOptions = {}) {
```

Comment:
FYI, I had to make all the changes around the signature of this function because Sonar was complaining.  I resisted the urge to do a broader refactor of this code (it could use it). However, I think combining the params and adding types actually did help improve the readability of this code... :+1: